src/collectd.conf.pod

```diff
@@ -2343,6 +2343,69 @@ Enabled by default, collects unknown (and thus presented as numeric only) query
 
 =back
 
+=head2 Plugin C<dpdkstat>
+
+The I<dpdkstat plugin> collects information about DPDK interfaces using the
+extended NIC stats API in DPDK.
+
+B<Synopsis:>
+
+ <Plugin "dpdkstat">
+    Interval 1
+    Coremask "0x4"
+    MemoryChannels "4"
+    ProcessType "secondary"
+    FilePrefix "rte"
+    EnabledPortMask 0xffff
+    PortName "interface1"
+    PortName "interface2"
+ </Plugin>
+
+B<Options:>
+
+=over 4
+
+=item B<Interval> I<seconds>
+
+The interval within which to retrieve stats in seconds. For milliseconds simple
+divide the time by 1000 for example if the desired interval is 50ms, set
+interval to 0.05.
+
+=item B<Coremask> I<Mask>
```

Comment:
If I read the code correctly, `Mask` is actually expected to be a string. The documentation should reflect that.